### PR TITLE
"Added index.html"

### PR DESCRIPTION
**Pull request by Verde9**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav>
+      <ul></ul>
+      <ul></ul>
+      <ul></ul>
+      <ul></ul>
```
Comment (jonly03) [inline index.html]: @Verde9 did you mean to have a `<ul>` and a bunch `<li>` inside
Also what are our menu items called?

**Review by jonly03 — CHANGES_REQUESTED** [review]
Left you some comments.
Otherwise, everything else LGTM

File: index.html (hunk 1 above)
Comment (Verde9) [reply index.html]: @jonly03 Just updated index.html, please review.

**Review by jonly03 — APPROVED** [review]
LGTM

**Pull request merged**